src/style.css

```diff
@@ -0,0 +1,97 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.box {
+  height: 300px;
+  min-width: 300px;
+
+  text-align: center;
+  line-height: 300px;
+  color: #fff;
+  font-size: 100px;
+}
+
+.box_1 {
+  background-color: #f00;
+}
+
+.box_2 {
+  background-color: #c00;
+}
+
+.box_3 {
+  background-color: #900;
+}
+
+.box_4 {
+  background-color: #600;
+}
+
+.box_5 {
+  background-color: #300;
+}
+
+.box_6 {
+  background-color: #000;
+}
+
+@media screen and (min-width: 900px) {
+  .box {
+    flex-basis: 33.33%;
+  }
```

Comment:
try to achieve the same effect without changing flex-basis

Reply:
Here I understand from slack that don't need to change.